drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -37,6 +37,18 @@ def client_search(input:, **args)
       resolve_clients(search_scope, **args)
     end
 
+    clients_field :client_omni_search, 'Client omnisearch' do |field|
+      field.argument :input, Types::HmisSchema::ClientSearchInput, required: true
+    end
+
+    def client_omni_search(input:, **args)
+      client_scope = Hmis::Hud::Client.client_search(input: input.to_params, user: current_user).
+        joins(:enrollments).
+        where(Enrollment: { id: Hmis::Hud::Enrollment.open_during_range((Date.today - 1.month)..Date.today) }).
+        order('Enrollment.DateUpdated' => :desc)
```

Comment:
@eanders I'm not sure that this is correct, but I figure I'd ask you before spending more time on it. I imagine you have a better idea of what needs to be here

Reply:
Resolving, as Elliot and I paired and fixed this